lib/mindwendel_web/live/idea_live/form_component.ex

```diff
@@ -27,6 +27,24 @@ defmodule MindwendelWeb.IdeaLive.FormComponent do
     save_idea(socket, socket.assigns.action, idea_params)
   end
 
+  defp save_idea(socket, :update, idea_params) do
+    idea = Brainstormings.get_idea!(idea_params["id"])
```

Comment:
what do you think about also querying for the user id to make sure that the idea is from the user? to prevent manipulation attempts by other users. the admin would something else though.

Reply:
I will look into this.

Reply:
Fixed for now. But I will need to tackle this later